meal-planning/src/components/shopping-list/ShoppingList.js

```diff
@@ -1,7 +1,70 @@
 import React from "react";
+import ReactDOM from 'react-dom'
+import { makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Checkbox from '@material-ui/core/Checkbox';
+import IconButton from '@material-ui/core/IconButton';
+import styles from "./ShoppingList.module.scss";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
we already have a global theme made in theme.js, you should put your styling in the .scss file if you want to change something in your component